pkg/block/block.go

```diff
@@ -95,3 +101,59 @@ func renameFile(from, to string) error {
 	}
 	return pdir.Close()
 }
+
+// DownloadDir downloads directory that is mean to be block directory (ends with ULID and contains block data).
+func DownloadDir(ctx context.Context, bucket objstore.Bucket, blockDir, dst string) error {
```

Comment:
`Download` or `DownloadBlock` I'd say. Otherwise two central pkgs have the same function name and signature, which seems rather confusing.

Reply:
Why? 
One is block aware, second object aware